### ISSUE TITLE
Add SSL/TLS support

### ISSUE THREAD

**Issue by wanderanimrod** [Cobliteam/cassandra-migrate#16]
Hello Cobliteam, thanks for your work on this tool. 

We have hit a snug trying to run migrations against a Cassandra cluster with `client_encryption` enabled. The migrations fail to connect to the cluster because cassandra-migrate is trying to connect without SSL.

```
WARNING:cassandra.cluster:[control connection] Error connecting to [IP_ADDRESS]:
Traceback (most recent call last):
  File "cassandra\cluster.py", line 2648, in cassandra.cluster.ControlConnection._reconnect_internal (cassandra/cluster.c:48586)
    return self._try_connect(host)
  File "cassandra\cluster.py", line 2670, in cassandra.cluster.ControlConnection._try_connect (cassandra/cluster.c:49103)
    connection = self._cluster.connection_factory(host.address, is_control_connection=True)
  File "cassandra\cluster.py", line 1111, in cassandra.cluster.Cluster.connection_factory (cassandra/cluster.c:15327)
    return self.connection_class.factory(address, self.connect_timeout, *args, **kwargs)
  File "cassandra\connection.py", line 330, in cassandra.connection.Connection.factory (cassandra/connection.c:5678)
    raise conn.last_error
ConnectionShutdown: Connection to [IP_ADDRESS] was closed
```

The cassandra-driver [supports SSL connections](http://datastax.github.io/python-driver/api/cassandra/cluster.html#cassandra.cluster.Cluster.ssl_options).  

We think a fix would entail passing `ssl_options` to the `Cluster` instantiation call [here](https://github.com/Cobliteam/cassandra-migrate/blob/63878e627712fe2ca296bb7b17da06104dd1c149/cassandra_migrate/migrator.py#L111), and then allowing users to provide an ssl certificate file path in the configuration yaml or as an environment variable passed to the `cassandra-migrate` command.

We are working on an enhancement PR to this effect but we will need your help publishing a new version pretty fast. 

Please let us know if you know of a better way to do this, and if you are open to the PR. 

Thanks.

**Comment by danielkza** [comment]
Your solution sounds completely reasonable. A PR would be very welcome, as I do not have a TLS-enabled Cassandra cluster to test with.

**Comment by wanderanimrod** [comment]
Thanks for the quick response @danielkza . A PR will be coming soon.

**Comment by danielkza** [comment]
Released in 0.2.5. Thanks!

**Comment by wanderanimrod** [comment]
Thanks a lot @danielkza for the quick response.
